gd/gd.php

```diff
@@ -3013,3 +3013,8 @@ function imagescale ($image, $new_width, $new_height = -1, $mode = IMG_BILINEAR_
  * @since 5.5
  */
 function imagesetinterpolation ($image, $method = IMG_BILINEAR_FIXED) {}
+
+/**
+ * @since 8.0
+ */
+final class GdImage{}
```

Comment:
Should we add a private constructor to enforce: `Cannot be instantiated with new \GdImage().` ?

Reply:
Decide yourself. I did it the same as `CurlHandler` – just a class definition.

Reply:
FYI: they aren't cloneable too: https://github.com/php/php-src/blob/master/ext/gd/tests/gdimage_prevent_cloning.phpt

Reply:
But CurlHandle *is* cloneable: https://github.com/php/php-src/blob/master/ext/curl/tests/curl_handle_clone.phpt (sure why not 🤣)